### PR TITLE
Issue 241

### PR DESCRIPTION
**Pull request by rsk2**
## Tests and linting

 - [ ] I have rebased my changes on `main`

 - [ ] `just lint` passes

 - [ ] `just test` passes

### CODE REVIEW

File: OpenOversight/app/email.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ def send_email(to, subject, template, **kwargs):
     msg.body = render_template(template + ".txt", **kwargs)
     msg.html = render_template(template + ".html", **kwargs)
     # Only send email if we're in prod or staging, otherwise log it so devs can see it
-    if app.env in ("staging", "production"):
+    if app.debug in ("staging", "production"):
```
Comment (rsk2) [inline OpenOversight/app/email.py:24]: Hi @sea-kelp, I have changed `app.env` to `app.debug`. Is this fine?

Comment (sea-kelp) [reply OpenOversight/app/email.py:24]: We actually want to use `app.config["ENVIRONMENT"]` to check the value of the new variable you've added!

**Review by sea-kelp — CHANGES_REQUESTED** [review]
Great work! There is just one other place where `FLASK_ENV` is used
https://github.com/OrcaCollective/OpenOversight/blob/1de4c6bd8963dc3bf3878df0439cac32effc41f1/.env.example#L12

File: OpenOversight/app/email.py (hunk 1 above)
Comment (rsk2) [reply OpenOversight/app/email.py:24]: Ok, makes sense

**Comment by sea-kelp** [comment]
Hi @rsk2, looks like there's one more place where you need to make a change:

This section:
https://github.com/OrcaCollective/OpenOversight/blob/1de4c6bd8963dc3bf3878df0439cac32effc41f1/OpenOversight/app/config.py#L11-L12

should be changed to:
```python
class BaseConfig(object):
    ENVIRONMENT = os.environ.get("ENVIRONMENT", "")
    # DB SETUP
```

**Comment by sea-kelp** [comment]
So close! The tests are failing because these two lines should have `app.config["ENV"]` changed to `app.config["ENVIRONMENT"]`

https://github.com/OrcaCollective/OpenOversight/blob/1de4c6bd8963dc3bf3878df0439cac32effc41f1/OpenOversight/app/commands.py#L534

https://github.com/OrcaCollective/OpenOversight/blob/1de4c6bd8963dc3bf3878df0439cac32effc41f1/OpenOversight/app/commands.py#L575

**Comment by rsk2** [comment]
@sea-kelp , I have made the changes in [OpenOversight/OpenOversight/app/commands.py](https://github.com/OrcaCollective/OpenOversight/blob/1de4c6bd8963dc3bf3878df0439cac32effc41f1/OpenOversight/app/commands.py#L534)

Additionally, I ran `just test` on my local and `test_commands` passes. However, a different test has failed; please see the screenshot below:

image

**Comment by sea-kelp** [comment]
> @sea-kelp , I have made the changes in [OpenOversight/OpenOversight/app/commands.py](https://github.com/OrcaCollective/OpenOversight/blob/1de4c6bd8963dc3bf3878df0439cac32effc41f1/OpenOversight/app/commands.py#L534)
> 
> Additionally, I ran `just test` on my local and `test_commands` passes. However, a different test has failed; please see the screenshot below:
> 
> image

It looks like the test run passed on here so I think it might just be a transient Selenium error?

Really appreciate your help with fixing this issue! :)

**Review by sea-kelp — APPROVED** [review]

**Comment by AetherUnbound** [comment]
Thank you for the contribution @rsk2!

**Comment by AetherUnbound** [comment]
I've updated our `.env` file in production so this won't cause any hiccups on our next release 🙂

**Pull request merged**